src/librustc_metadata/rmeta/encoder.rs

```diff
@@ -693,16 +693,23 @@ impl EncodeContext<'tcx> {
         vis: &hir::Visibility<'_>,
     ) {
         let tcx = self.tcx;
-        let def_id = tcx.hir().local_def_id(id).to_def_id();
+        let def_id = tcx.hir().local_def_id(id);
         debug!("EncodeContext::encode_info_for_mod({:?})", def_id);
 
         let data = ModData {
             reexports: match tcx.module_exports(def_id) {
-                Some(exports) => self.lazy(exports),
+                Some(exports) => {
+                    let map = self.tcx.hir();
+                    self.lazy(
+                        exports.iter().map(|export| export.map_id(|id| map.as_local_hir_id(id))),
+                    )
```

Comment:
Lots of `map` here! Maybe `hir` would be a better name for the local variable?
```suggestion
                    let hir = self.tcx.hir();
                    self.lazy(
                        exports.iter().map(|export| export.map_id(|id| hir.as_local_hir_id(id))),
                    )
```

Reply:
It's called the hir map and `map` is used in different places as a variable name to denote the hir map.

Reply:
I opted for `hir_map`.